src/lib/transaction_snark/test/zkapps_examples/empty_update/empty_update.ml

```diff
@@ -0,0 +1,165 @@
+open Transaction_snark_tests.Util
+open Core_kernel
+open Mina_base
+open Signature_lib
+module Impl = Pickles.Impls.Step
+module Inner_curve = Snark_params.Tick.Inner_curve
+module Nat = Pickles_types.Nat
+module Local_state = Mina_state.Local_state
+module Parties_segment = Transaction_snark.Parties_segment
+module Statement = Transaction_snark.Statement
+
+let sk = Private_key.create ()
+
+let pk = Public_key.of_private_key_exn sk
+
+let pk_compressed = Public_key.compress pk
+
+let account_id = Account_id.create pk_compressed Token_id.default
+
+let tag, _, p_module, Pickles.Provers.[ prover; _ ] =
```

Comment:
could you put everything inside `%test_module`? or maybe a functor if it is going to be reused in other tests so that proof generation and tests are not in toplevel

Reply:
This is actually an executable (which is added to the `runtest` alias, so it still gets run with tests). The intention was that this can be run to dump the `Parties.t` to be able to send to a network with minimal changes.

Reply:
Sounds good. I didn't notice that it was an executable.